lib/validateMessage.js

```diff
@@ -21,6 +21,45 @@ var error = function() {
 
 exports.config = config;
 
+var validateScope = function(isValid, scope) {
```

Comment:
And actually, because this is a utility method, I prefer that it go at the bottom of the file. Read more about my preference [here](https://medium.com/@kentcdodds/newspaper-code-structure-8b4616af9539)